### PR TITLE
Fix #481: Tick slider not snapping if clicked

### LINKED ISSUE #481: Tick slider not snapping to ticks if clicked
When dragging the tick slider works just fine, but when you click in between ticks, the value will be of that in between - its not snapping the way it is when you're dragging. And that is what I would think that should happen.

JSFiddle: http://jsfiddle.net/h3WDq/995/

### PR DESCRIPTION
**Pull request by jasonkonopacki**
Let me know if you need anything else. Issue #481 

Added _snapToValue(val) when calling _setDataVal and ._trigger('slideStop', in the _mouseup function.

### CODE REVIEW

**Comment by seiyria** [comment]
As per our [contribution guidelines](https://github.com/seiyria/bootstrap-slider/blob/master/.github/CONTRIBUTING.md) and [PR checklist](https://github.com/seiyria/bootstrap-slider/blob/master/.github/PULL_REQUEST_TEMPLATE.md), we'll need:

* unit tests (so this doesn't happen again)
* a before/after jsfiddle clearly demonstrating the issue

File: dist/bootstrap-slider.js [hunk 1]
```diff
@@ -1376,8 +1376,8 @@ function _typeof(obj) { return obj && typeof Symbol !== "undefined" && obj.const
 				var val = this._calculateValue(true);
```
Comment (rovolution) [inline dist/bootstrap-slider.js]: this is a generated file. please make your change in the `/src` file.

Also...where does this `_snapToValue()` function come from? I dont see it defined anywhere in our source code

**Comment by jasonkonopacki** [comment]
The issue appears to only happen with touch events and the fix is spotty. Will reopen when fixed.

File: dist/bootstrap-slider.js (hunk 1 above)
Comment (jasonkonopacki) [inline dist/bootstrap-slider.js]: Wow, I totally missed on this one.

I was using dandrel's comment fix from #392 along with my new code.

Sorry about that guys, a bit new to github and lost track of myself making that change.

**Pull request closed without merging**